docs/source/develop/api/permissions.rst

```diff
@@ -1172,7 +1172,7 @@ Allows getting any `transaction <../../before_start/glossary.html#transaction>`_
 .. literalinclude:: ../../../../example/python/permissions/can_get_all_txs.py
```

Comment:
docs/source/develop/api/permissions.rst is an autogenerated file. 
This needs to be fixed in /example/python/permissions/can_get_all_txs.py

Reply:
I ran `make permissions` and got this file. The file that is currently in the repository is outdated.

Reply:
My bad, I thought that permissions were made during the build process